packages/wheelhouse-frontend/src/components/reusables/DataContainer.js

```diff
@@ -1,4 +1,4 @@
-import React from "react"; 
+import React from "react";
```

Comment:
Hmm, looks like ESLint or VS Code or something made some whitespace changes in this file? Strange. I'm cool with it though.

@megelismi: You should check your Sublime settings and make sure it's set to strip extra whitespace from the ends of lines, looks like a couple errant spaces got committed here